src/components/SchemaRow/SchemaRow.tsx

```diff
@@ -67,6 +68,14 @@ export const SchemaRow: React.FunctionComponent<SchemaRowProps> = React.memo(
     const validations = isRegularNode(schemaNode) ? schemaNode.validations : {};
     const hasProperties = useHasProperties({ required, deprecated, validations });
 
+    React.useEffect(() => {
```

Comment:
Couldn't think of a better way to ensure the internal state of the row is in sync